graphene/src/intersection.rs

```diff
@@ -10,14 +10,17 @@ use glam::{DAffine2, DMat2, DVec2};
 use kurbo::{BezPath, CubicBez, Line, ParamCurve, ParamCurveExtrema, PathSeg, Point, QuadBez, Rect, Shape, Vec2};
 
 #[derive(Debug, Clone, Default, Copy)]
+/// A quadriliteral defined by four vertices.
```

Comment:
"rectangle" would be clearer

Reply:
hmmm is it a rectangle? i dont think a quad necessarily has 90° corners

Reply:
Oh, you're right. My bad. Then I'd suggest calling it "quad" then since the full word, quadrilateral, is not very common. (It's also misspelled at the moment, but easier to just shorten it to its more common form.)